src/hotspot/cpu/zero/methodHandles_zero.cpp

```diff
@@ -208,6 +208,7 @@ address MethodHandles::generate_method_handle_interpreter_entry(MacroAssembler*
   switch (iid) {
   case vmIntrinsics::_invokeGeneric:
   case vmIntrinsics::_compiledLambdaForm:
+  case vmIntrinsics::_linkToNative:
```

Comment:
Sorry, another issue. What is this? Do we need this to compile Zero? Does it even make sense to have it for `method_handle_entry`?

Reply:
This function gets called during the build. If the case fro linkToNative is not added it falls through to ShouldNotReachHere() in the default case.

Pointing it to method_handle_entry_invalid seemed like the best way to fix it, since the interpreter entry never gets used any ways.

Reply:
(Though, I suppose in general the entry never gets used because we don't have a CLinker implementation for Zero)

Reply:
Then I suggest doing what `ZeroInterpreterGenerator::generate_abstract_entry` does and generate the entry that "calls" `ShouldNotCallThisEntry()`. Which means the build would succeed, but the attempt to actually call through `_linkToNative` would fail in a proper place.

Reply:
In fact, I think right before this method, there is `method_handle_entry_invalid` that you can target for `_linkToNative` case.

Reply:
...which you already use! Argh! Good then. Please disregard this noise. Ship it.